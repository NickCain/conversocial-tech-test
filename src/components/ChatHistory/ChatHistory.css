.Chat-history {
    flex-grow: 1;
    flex-shrink: 1;
}
