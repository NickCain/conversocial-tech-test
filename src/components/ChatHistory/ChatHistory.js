import PropTypes from 'prop-types';
import React, { Component } from 'react';

import styles from './ChatHistory.scss';

class ChatHistory extends Component {
  constructor(props) {
    super(props);

    this.state = {};
  }

  render() {
    return (
      <div />
    );
  }
}

ChatHistory.propTypes = {};

ChatHistory.defaultProps = {};

ChatHistory.contextTypes = {};

export default ChatHistory;
