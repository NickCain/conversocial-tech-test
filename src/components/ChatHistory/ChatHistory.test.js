import { mount, shallow } from 'enzyme';
import React from 'react';
import ChatHistory from './ChatHistory.js'

let wrapper;
let shallowWrapper;

describe('ChatHistory - Component', () => {
  describe('ChatHistory renders', () => {
    beforeEach(() => {
      wrapper = mount(
        <ChatHistory />
      );
      shallowWrapper = shallow(
        <ChatHistory />
      );
    });

    it('should render the ChatHistory component', () => {
      expect(shallowWrapper).toMatchSnapshot();
      expect(wrapper.length).toBeGreaterThan(0);
    });
  });
});
