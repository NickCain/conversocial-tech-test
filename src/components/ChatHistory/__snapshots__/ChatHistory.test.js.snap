// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatHistory - Component ChatHistory renders should render the ChatHistory component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="Chat-history"
>
    <h3>
        Chat History
    </h3>
</div>,
  "nodes": Array [
    <div
      className="Chat-history"
>
      <h3>
            Chat History
      </h3>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <ChatHistory />,
      "_debugID": 5,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": ChatHistory {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="Chat-history"
>
          <h3>
                    Chat History
          </h3>
</div>,
        "_debugID": 6,
        "_renderedOutput": <div
          className="Chat-history"
>
          <h3>
                    Chat History
          </h3>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ChatHistory />,
}
`;
